<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Obrazek 1</string>
    </property>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>72</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>130</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Obrazek 2</string>
    </property>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>72</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>210</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Obrazek 3</string>
    </property>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>99</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>72</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>290</y>
      <width>441</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Obrazek 4</string>
    </property>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_4">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>0</number>
     </property>
     <property name="value">
      <number>99</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>72</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
